madmom/audio/ffmpeg.py

```diff
@@ -295,7 +295,10 @@ def get_file_info(infile, cmd='ffprobe'):
         if line.startswith(b'channels='):
             info['num_channels'] = int(line[len('channels='):])
         if line.startswith(b'sample_rate='):
-            info['sample_rate'] = int(line[len('sample_rate='):])
+            if cmd == 'avprobe':
+                info['sample_rate'] = int(float(line[len('sample_rate='):]))
```

Comment:
Yes, the single line would probably also work, although I can't test it with ffmpeg.
If we convert it to one line, we should add a comment explaining why the `int(float(...))` thingy is necessary (because of avprobe)